makefiles/tools/mspdebug.inc.mk

```diff
@@ -1,17 +1,16 @@
-MSPDEBUGFLAGS += -j $(MSPDEBUG_PROGRAMMER)
-ifeq ($(strip $(MSPDEBUG_PROGRAMMER)),uif)
-  MSPDEBUGFLAGS += -d $(PROG_DEV)
-endif
```

Comment:
Apparently it's possible to flash using mspdebug over a TTY, using `uif-bsl` (maybe before it was called `uif` ?). Even if unused (no msp430 based board supports this, maybe it was for msp430 that were removed a few years ago ?), maybe we should keep it for compatibility ?

Reply:
`mspdebug` supports flashing via TTY instead of USB for a number of programmers, but defaults to using USB. My guess is that this is a workaround that I don't think is needed anymore. At least judging from the output of `mspdebug uif` it seems it would just select the matching USB device directly without a TTY given:

```
$ mspdebug uif   
MSPDebug version 0.25 - debugging tool for MSP430 MCUs
Copyright (C) 2009-2017 Daniel Beer <dlbeer@gmail.com>
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Chip info database from MSP430.dll v3.3.1.4 Copyright (C) 2013 TI, Inc.

usbutil: unable to find a device matching 0451:f430
```

But that theory will be difficult to put to test:

> The MSP-FET430UIF is no longer recommended for new designs.

And it seems to not be available any more. I have the feeling that most people actually taking a look at MSP430 will either use the new boards with FRAM with the integrated eZ-FET programmer/debugger (once that is ported to RIOT), or go for Olimex hardware with legacy MSP430Fx1xx hardware. Maybe @OlegHahm has still legacy programmers at hands and could give it a test?